### ISSUE TITLE
Remove "cache_" prefix

### ISSUE THREAD

**Issue by infabo** [AOEpeople/TYPO3_Restler#81]
```
Core: Error handler (BE): TYPO3 Deprecation Notice: Accessing a cache with the "cache_" prefix as in "restler" is not necessary anymore, and should be called without the cache prefix. in /var/www/html/vendor/typo3/cms-core/Classes/Cache/CacheManager.php line 112 "
```

I am using not the default cache-backend for EXT:restler, namely NullBackend locally and RedisBackend on production.
When I open the Install Tool, the automatic migration rewrites the the cache-name from `cache_restler` to just `restler`.

As I am using helhum config handling with YAML config, the auto migration can't complete because the wizard can't modify YAML files and the migration ends in a loop.

My suggestion: Please adjust all your usages of "restler" to `restler` like:

https://github.com/AOEpeople/TYPO3_Restler/blob/11.0.6/ext_localconf.php#L48
and
https://github.com/AOEpeople/TYPO3_Restler/blob/11.0.6/ext_localconf.php#L55

Open for discussion.

TYPO3 11.5.30

**Comment by infabo** [comment]
The `cache_`-prefix is not even allowed in v11. We need a fix for this please. 

https://github.com/AOEpeople/TYPO3_Restler/blob/11.0.6/ext_localconf.php#L50

**Comment by soda-2005** [comment]
Hi infabo,
we fixed the bug / deprecation now in version 11.2.0 :-)
